Cargo.toml

```diff
@@ -24,7 +24,7 @@ crypto-hash = "0.3"
 curl = "0.4.6"
 docopt = "0.8.1"
 env_logger = "0.4"
-error-chain = "0.11.0"
+failure = "0.1"
```

Comment:
You use `bail!`, so this should be `0.1.1` in case some monster tries to say `failure = "=0.1.0"`.